### ISSUE TITLE
Missing stone chunks

### ISSUE THREAD

**Issue by 4c5458** [webbukkit/dynmap#3559]
**Issue Description:** *Custom world render misinterpretates stone Y level. It looks like chunks that contain only stone are rendered incorrectly.*

World looks correctly on 1.17:
![Screenshot_1 17](https://user-images.githubusercontent.com/47794449/145718712-bff1d807-daea-49c1-aab5-5c24ccf37d28.png)

World looks weird on 1.18:
![Screenshot_1 18](https://user-images.githubusercontent.com/47794449/145718717-b701df3e-2785-4380-a9bd-8a6e9487f341.png)

I've tried:
- re-render completly several times
- placing blocks, chunk appears, deleting blocks, chunk vanishes again
- Worldeditting surfaces with stone again.

* **Dynmap Version:** *dynmap v3.3-SNAPSHOT-573*
* **Server Version:** *Paper version git-Paper-71 (MC: 1.18.1)*
* **Pastebin of Configuration.txt:** *https://pastebin.com/ekAjN4Fx*
* **Server Host (if applicable):** *NA*
* **Pastebin of crashlogs or other relevant logs:** *none*
* **Other Relevant Data/Screenshots:** *World setting: https://pastebin.com/GPHtit4R*
* **Steps to Replicate:** *Setup test server, default config, world settings: see above. Fullrender, see result below*

 [X] *I have looked at all other issues and this is not a duplicate*  
 [X] *I have been able to replicate this*

**Comment by generrosity** [comment]
Hi - sorry, you replicated this with test server but the same settings files? Could you confirm with completely fresh?

This very much looks like how the nether ignores certain blocks. Or, if there are custom / modded blocks as well as normal blocks. I assume that you have looked at eg the blocks on the inside of the donut and they are the 'same' as the blocks on the top of the donut?
![image](https://user-images.githubusercontent.com/12838061/145915727-4a7d8c77-b193-4773-81c5-798a700d6016.png)

Check out the download area as there is a beta2 to try as well. As always, 1.18 isnt released and expected to be buggy at this stage

**Comment by 4c5458** [comment]
Yes, I did replicate this with the defaullt (empty) world file on a plain test server.
The result was this:
![Screenshot_default1 18](https://user-images.githubusercontent.com/47794449/145977863-f73f7706-2ff1-4192-9403-408c4dbd1240.png)

Meanwhile beta 2 is out. Tried that version too.

I tried to replace the stone surface (at y= 64) with stone. No result.
I tried to replace with a different kind of block. Now it shows. But when I changed it back to stone, it disappears again.
When I remove 1 block of the surface, that chunk appears (only the upper 16 levels).
Multiverse shows this world as NORMAL so nothing like nether.
The biome is `the_void`. As it always was in 1.17. Changing biome does not change render.

TL;DR: Any full stone chunk surface on y = n x 16 (where n = an integer) will disappear and I have no clue why

**Comment by mikeprimm** [comment]
Issue was with 'zero bit palette' handling (that is, how NBT for 1.18 chunks are encoded when a whole section is one block state).   SNAPSHOT #580 or later should have fix.

**Comment by 4c5458** [comment]
Works as intended again. Awsome.
